### DISCUSSION TITLE
Are there any plans for webssh to support using a key agent?

### DISCUSSION THREAD

**Discussion by ivahos** [isontheline/pro.webssh.net#1129] [General]
Hi, I am using 1password and it is. functioning as a ssh keyserver by setting the environment variable SSH_AUTH_SOCK and by placing IdentityAgent "~/Library/Group Containers/2BUA8C4S2C.com.1password/t/agent.sock" in the ssh config file in ~/.ssh

Are there any plans for adding support for using this to serve keys to remote hosts instead of storing them in webssh?

/var

**Comment by isontheline** [comment]
Hi @ivahos 👋 

Unfortunately Agent Forwarding is not - yet - available : https://github.com/isontheline/pro.webssh.net/issues/754

I have two concerns about the implementation : 
* not so trivial as it sounds 
* agent forwarding could be potentially be used by an attacker on the server side

But I will do my best to implement it.

**Reply by ivahos** [reply]
I am not talking about forwarding it to the host I am logging into, but for webssh to fetch the key from the local keyserver on my mac so that I don't have to store the keys in webssh but keep them in my password manager

/var

**Reply by isontheline** [reply]
Sorry 😅

I will check but as WebSSH is sandboxed it can’t access any file outside its own container.

I’m searching if 1 Password allows http retrieval
